life_expectancy/tests/test_cleaning.py

```diff
@@ -0,0 +1,16 @@
+"""Tests for the cleaning module"""
+import pandas as pd
+
+from life_expectancy.cleaning import main
+from . import OUTPUT_DIR
+
+
+def test_main(pt_life_expectancy_expected):
+    """Run the `clean_data` function and compare the output to the expected output"""
```

Comment:
you could also maybe add here the pt_life_expectancy_expected as Args in the docstring

Reply:
yes, will update it!